app/build.gradle

```diff
@@ -116,6 +116,10 @@ android {
             applicationIdSuffix ".debug"
             versionNameSuffix "-debug"
             signingConfig signingConfigs.debug
+
+            buildConfigField "String", "IP_SIMPLE_API_ACCESS_KEY", "\"AIzaSyBhJJeiBWKTfbV446m2sgJCP-EogEWZUh0\""
```

Comment:
This part is the debug buildType where arbitrary but valid keys are used. Should I move these to fields?